spacecmd/src/spacecmd/configchannel.py

```diff
@@ -847,6 +847,7 @@ def help_configchannel_updateinitsls(self):
 
 options:
   -c CHANNEL
+  -r REVISION
```

Comment:
It would be cool to have a (Cucumber?) test to cover this.

Reply:
Sure! I'll add a Cucumber test for this.

Reply:
@moio I've now updated this PR with a Cucumber test that will guarantee `-r REVISION` option is working properly.

Reply:
You have a green light from my side! Thanks!